wsmaster/che-core-sql-schema/src/main/resources/che-schema/7.19.0/1__userdevfile.sql

```diff
@@ -0,0 +1,29 @@
+--
+-- Copyright (c) 2012-2020 Red Hat, Inc.
+-- This program and the accompanying materials are made
+-- available under the terms of the Eclipse Public License 2.0
+-- which is available at https://www.eclipse.org/legal/epl-2.0/
+--
+-- SPDX-License-Identifier: EPL-2.0
+--
+-- Contributors:
+--   Red Hat, Inc. - initial API and implementation
+--
+
+
+-- add userdevfile table
+CREATE TABLE userdevfile (
```

Comment:
We're past `7.19.0`

Reply:
We're passing ) Master still has 7.19.0-SNAPSHOT